microstream-rest/src/test/groovy/io/micronaut/microstream/rest/StorageManagerListenerFuncTest.groovy

```diff
@@ -0,0 +1,62 @@
+package io.micronaut.microstream.rest
+
+import ch.qos.logback.classic.Logger
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.core.read.ListAppender
+import io.micronaut.context.ApplicationContext
+import io.micronaut.inject.qualifiers.Qualifiers
+import one.microstream.storage.types.StorageManager
+import org.slf4j.LoggerFactory
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.TempDir
+
+class StorageManagerListenerFuncTest extends Specification {
```

Comment:
I think we should remove this test and the listener.

Reply:
done